### DISCUSSION TITLE
How to connect existing User to Post

### DISCUSSION THREAD

**Discussion by cakctus** [prisma/prisma#16988] [Q&A] [answered]
const user = await prisma.user.findUnique({
    where: {
      id: userId,
    },
  })

  const modification = await prisma.cars_modification.findUnique({
    where: {
      id: modificationId,
    },
  })

  const commentInstance = await prisma.comment.create({
    data: {
      comment,
      userId: user.id,
      cars_modificationId: modification.id,
    },
  })

Result: 
![Screenshot_1](https://user-images.githubusercontent.com/39764706/209173968-6ec4b6ca-3a78-4c38-b1a3-fba2293586bb.png)
In my result userId is just a number but i want user instance.

**Comment by nurul3101** [comment, marked as answer]
Hello @cakctus 👋 

It looks like you would need to use the `include` clause to get the user details: [Include Reference](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#include)
Can you try adding an include clause in your query?

**Reply by cakctus** [reply]
Thanks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
